packages/client/src/Components/Mapping/helpers/MapPlanningPlayerListener.js

```diff
@@ -78,13 +79,10 @@ export default class MapPlanningPlayerListener {
     // command to submit whole planned route
     const context = this
```

Comment:
Is `context` needed here or can you refactor it out?

Reply:
Just tried - yes, I can lose it.  I believe there are still occasions when it's required - typically in Leaflet click callbacks, I think.